rb/Steepfile

```diff
@@ -1,8 +1,76 @@
-# frozen_string_literal: true
-
 target :lib do
-  signature 'sig' # Signature directory
+  signature 'sig', '.gem_rbs_collection/rubyzip' # Signature directory
   check 'lib' # Directory name
+  # Total amount of errors ignore 61 in 27 files
+  ignore(
+    # Ignore line 166 due to UDP RBS issue
+    'lib/selenium/webdriver/common/platform.rb',
```

Comment:
Is it possible to disable steep inside the file itself? Something similar to `# rubocop:disable` comment.

Reply:
I haven't found anything online but I asked on the discord server, if I get the confirmation that is not possible I will create a PR in the steep project because it will be useful

Reply:
I got some help on Discord and there is a merged PR for the release 1.7.0 in steep that adds support for this, but the release is not ready yet, so for now this is the only way to ignore

https://github.com/soutaro/steep/pull/1034

Reply:
We could just install steep from GitHub via Gemfile directly to avoid waiting for release.

```ruby
gem 'steep', github: 'soutaro/steep', require: false, platforms: %i[mri mingw x64_mingw]
```

Reply:
I tried this, and I followed up the new way to ignore per line, but I cannot make it work, I will try tomorrow and reach out again in discord, but it seemed with the steep release 1.7.0-dev2 I couldn't make it work

Reply:
I will make a PR updating the ignore files when release 1.7.0 is released and add the lines that should be ignored